buildchain/buildchain/iso.py

```diff
@@ -126,6 +126,21 @@ def task__iso_render_bootstrap() -> types.TaskDict:
     ).task
 
 
+def task__add_upgrade() -> type.TaskDict:
+    """Copy the Upgrade script to script folder."""
+    src = constants.ROOT/'scripts'/'upgrade.sh'
+    dest = constants.ROOT/'upgrade.sh'
+    copying_upgrade = [src, dest]
+    return{
+        'title': lambda task: utils.title_with_target1('COPY', task),
+        'actions': [(coreutils.cp_file, upgrade)],
+        'targets': [dest],
+        'task_dep': [__iso_mkdir_root],
+        'file_dep': [src],
+        'uptodate': [True],
```

Comment:
Nope.
Why do you want it to be always true?

If the script is modified, you want to copy it again.